scripts/semconv/templates/registry/weaver.yaml

```diff
@@ -13,7 +13,7 @@ templates:
     filter: >
       semconv_grouped_attributes({
         "exclude_root_namespace": $excluded_namespaces,
-        "exclude_stability": if $filter == "any" then [] else ["experimental"] end,
+        "exclude_stability": if $filter == "any" then [] else ["experimental", "", null] end,
```

Comment:
Should the same be done for the `exclude_stability` below for `pattern: semantic_metrics.j2`?

Reply:
yep, fixed